template_input.yaml

```diff
@@ -54,13 +54,40 @@ rules:
         operation: 
         input_variables: [,]
         output_variable: 
+ 
+  - formula_rule:
+      name: 
+      description:
+      formula:
+      input_variables: [,]
+      output_variable:
 
-  - combine_results_rule:
```

Comment:
why is this removed?

Reply:
they thought this rule was so nice, they added it twice ;)

Reply:
:)